ansible/tasks/system.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hostname:
+    name: "{{ pi_hostname }}"
+
+- locale_gen:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - "nl_NL.UTF-8"
```

Comment:
Maybe use English, for consistency with our other Linux systems?

Reply:
In that case we can simply omit the locale generation, right?

Reply:
That depends on the distro I think. On Ubuntu we needed [this](https://github.com/svsticky/sadserver/blob/626c97ec0d94ccb49d1f5b43ef29bcf1d980eb25/ansible/tasks/system-settings.yml#L43) to prevent warnings of unset locale-related environment variables.